spec/helpers/users_helper_spec.rb

```diff
@@ -0,0 +1,6 @@
+module UsersHelper
+
+  def is_logged_in?
+    !session[:user_id].nil?
```

Comment:
is this an actual test? what is it doing?

Reply:
no, it is used in tests to check if a user is logged in